e2e/test/scenarios/models/reproductions/29951-model-editor-results-metadata.cy.spec.js

```diff
@@ -15,35 +17,39 @@ const questionDetails = {
       CC1: ["+", ["field", ORDERS.TOTAL], 1],
       CC2: ["+", ["field", ORDERS.TOTAL], 1],
     },
+    limit: 5,
   },
   dataset: true,
 };
 
-describe("issue 29951", () => {
+describe("issue 29951", { requestTimeout: 10000 }, () => {
   beforeEach(() => {
     restore();
     cy.signInAsAdmin();
-    cy.intercept("POST", "/api/dataset").as("dataset");
     cy.intercept("PUT", "/api/card/*").as("updateCard");
+    cy.intercept("GET", `/api/database/${SAMPLE_DB_ID}/schema/PUBLIC`).as(
+      "publicShema",
+    );
   });
 
   it("should allow to run the model query after changing custom columns (metabase#29951)", () => {
     cy.createQuestion(questionDetails, { visitQuestion: true });
-    cy.wait("@dataset");
 
     openQuestionActions();
-    // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-    cy.findByText("Edit query definition").click();
+    popover().findByText("Edit query definition").click();
+    cy.wait("@publicShema");
```

Comment:
These two lines were the source of the merge conflict from https://github.com/metabase/metabase/pull/32239/files#diff-4f3b89aa6f745bf9fa183a2c7f288a1f391232932bf17a25ff15fc41ce50e932R36